vtm-android-example/src/org/oscim/android/test/VectorLayerActivity.java

```diff
@@ -67,26 +73,52 @@ public void onCreate(Bundle savedInstanceState) {
         //        }
         //    }
 
+        addRandomCircles(vectorLayer);
+        addThickSemitransparentPolyline(vectorLayer);
+
+        vectorLayer.update();
+
+        mMap.layers().add(vectorLayer);
+    }
+
+    private void addRandomCircles(VectorLayer vectorLayer) {
         Style.Builder sb = Style.builder()
-                .buffer(0.5)
-                .fillColor(Color.RED)
-                .fillAlpha(0.2f);
+            .buffer(0.5)
+            .fillColor(Color.RED)
+            .fillAlpha(0.2f);
```

Comment:
The formatting has changed?
We use the default IDEA settings.

Reply:
Sorry, I will correct this